### ISSUE TITLE
add details on how to verify kubernetes running successfully

### ISSUE THREAD

**Issue by anuaimi** [upmc-enterprises/kubeadm-aws#3]
I've followed the script and things seem to be working.  AWS shows the 3 VMs running.  I can log into the master and use kubeadm.  But when I try to follow the smoke-test instructions from Kubernets-the-hard-way, it doesn't work.   Can this tool be update to allow the smoke test to work

https://github.com/kelseyhightower/kubernetes-the-hard-way/blob/master/docs/09-smoke-test.md

I believe the issue is that the smoke test expects some tags to be set on the VMs

**Comment by anuaimi** [comment]
I was able to get the smoke test working by making two changes to the instructions...

look for `allow_ssh` security group rather than `kubernetes` 

```
export SECURITY_GROUP_ID=$(aws ec2 describe-security-groups \
  --filters "Name=tag:Name,Values=allow_ssh" | \
  jq -r '.SecurityGroups[].GroupId')
```

and use the '[TF] k8s-worker1' tag rather than 'worker0' tag to get the Node's public IP.

```
export NODE_PUBLIC_IP=$(aws ec2 describe-instances \
  --filters "Name=tag:Name,Values='[TF] k8s-worker1'" | \
  jq -j '.Reservations[].Instances[].PublicIpAddress')
```

**Comment by stevesloka** [comment]
Resolved with #5
